pkg/kubelet/kubelet_volumes.go

```diff
@@ -39,17 +39,16 @@ func (kl *Kubelet) ListVolumesForPod(podUID types.UID) (map[string]volume.Volume
 	return volumesToReturn, len(volumesToReturn) > 0
 }
 
-// podVolumesExist examines the directory structure for a pod and returns
-// true if it contains any mounted volumes. An error is returned if unable to
-// read directory.
-func (kl *Kubelet) podVolumesExist(podUID types.UID) (bool, error) {
+// podVolumesExist checks wiht the volume manager and returns true any of the
```

Comment:
`checks with the volume manager and returns true if any of the volumes for the specified pod are mounted`